bin/scancli

```diff
@@ -17,6 +17,7 @@
 # along with IVRE. If not, see <http://www.gnu.org/licenses/>.
 
 from ivre import utils, db, graphroute, config, xmlnmap
+from ivre.analyzer import Context
```

Comment:
Can you `try` ... `except ImportError` this and set a global variable like `USE_ANALYZER`? This would make new dependencies "soft" (use `git grep USE_ bin/ ivre/` for examples).